pkg/test/resourcefixture/testdata/basic/compute/v1beta1/computeforwardingrule/regionalcomputeforwardingrule/_generated_object_regionalcomputeforwardingrule.golden.yaml

```diff
@@ -34,9 +19,7 @@ spec:
     addressRef:
       name: computeraddress-${uniqueId}
   ipProtocol: ESP
-  loadBalancingScheme: EXTERNAL
```

Comment:
I think this is OK, this is a result of state-into-spec: absent and is maybe a candidate for adding to observedState